react/Modal/Readme.md

```diff
@@ -34,6 +47,25 @@ const sizes = [
 </div>
 ```
 
+### Space
+
+Besides the default spacing inside a Modal, you can choose another type from this two avalaible: `compressed` and `spacious`
```

Comment:
I don't understand when I would choose the spacious mode ?

Can you give some guidelines @Claiw ? I think it would be good to add them to the styleguide.

Reply:
For now, in the styleguide we tell what is possible. It is easy and already helpful but I think what would be even more useful would be to guide the user towards the right solution.  Here, if I were to read this, I would think "5 sizes and 3 possibles spacing mode ? That gives me 15 different possibilities to choose from when I am designing the modal. How can I choose ?".

Do you have any idea @Claiw on how to improve this ?

I guess what I'm trying to say here is that I feel we start to have too much options.

Reply:
This was indeed complicated to code but the Design Team seemed to need this.
Although I agree with you, maybe there should be a very highlighted default solution and let those options… very optional.